src/apps/Altinn.AccessManagement/src/Altinn.AccessMgmt.Persistence/Data/DbDataMigrationService.cs

```diff
@@ -1208,8 +1208,8 @@ public async Task IngestPackage(CancellationToken cancellationToken = default)
-            new Package() { Id = Guid.Parse("2f176732-b1e9-449b-9918-090d1fa986f6"), ProviderId = digdirProvider, EntityTypeId = orgEntityType, AreaId = area_fullmakter_for_revisor, Urn = "urn:altinn:accesspackage:ansvarlig-revisor", Name = "Ansvarlig revisor", Description = "Denne fullmakten gir revisor tilgang til å opptre som ansvarlig revisor for en kunde og utføre alle tjenester som krever denne fullmakten. Dette er tjenester som tjenestetilbyder har vurdert det som naturlig at en revisor utfører på vegne av sin kunde. Fullmakten gis kun til autoriserte revisorer. Fullmakt hos revisor oppstår når kunden registrerer regnskapsfører i Enhetsregisteret. Ved regelverksendringer eller innføring av nye digitale tjenester kan det bli endringer i tilganger som fullmakten gir.", IsDelegable = true, HasResources = true },
-            new Package() { Id = Guid.Parse("96120c32-389d-46eb-8212-0a6540540c25"), ProviderId = digdirProvider, EntityTypeId = orgEntityType, AreaId = area_fullmakter_for_revisor, Urn = "urn:altinn:accesspackage:revisormedarbeider", Name = "Revisormedarbeider", Description = "Denne fullmakten gir revisor tilgang til å opptre som revisormedarbeider for en kunde og utføre alle tjenester som krever denne fullmakten. Dette er tjenester som tjenestetilbyder har vurdert det som naturlig at en revisor utfører på vegne av sin kunde. Fullmakten gis kun til autoriserte revisorer. Fullmakt hos revisor oppstår når kunden registrerer regnskapsfører i Enhetsregisteret. Ved regelverksendringer eller innføring av nye digitale tjenester kan det bli endringer i tilganger som fullmakten gir.", IsDelegable = true, HasResources = true },
+            new Package() { Id = Guid.Parse("2f176732-b1e9-449b-9918-090d1fa986f6"), ProviderId = digdirProvider, EntityTypeId = orgEntityType, AreaId = area_fullmakter_for_revisor, Urn = "urn:altinn:accesspackage:ansvarlig-revisor", Name = "Ansvarlig revisor", Description = "Denne fullmakten gir revisor tilgang til å opptre som ansvarlig revisor for en kunde og utføre alle tjenester som krever denne fullmakten. Dette er tjenester som tjenestetilbyder har vurdert det som naturlig at en revisor utfører på vegne av sin kunde. Fullmakten gis kun til autoriserte revisorer. Fullmakt hos revisor oppstår når kunden registrerer revisor i Enhetsregisteret. Ved regelverksendringer eller innføring av nye digitale tjenester kan det bli endringer i tilganger som fullmakten gir.", IsDelegable = true, HasResources = true },
```

Comment:
Må det ikke incrementeres ett versjonsnr for at oppdateringene skal bli tatt i miljøer med eksisterende deploy?

Reply:
Yep! Bumpet fra 3 til 4 i NeedMigration og LogMigration